sdk/cognitivelanguage/ai-language-text/README.md

```diff
@@ -445,15 +481,96 @@ async function main() {
 main();
 ```
 
+### Extractive Summarization
+
+Extractive summarization identifies sentences that summarize the article they belong to.
+
+```javascript
+const { AzureKeyCredential, TextAnalysisClient } = require("@azure/ai-language-text");
+const client = new TextAnalysisClient("<endpoint>", new AzureKeyCredential("<API key>"));
+const documents = [
+  "<long article>"
+];
+async function main() {
+  const actions = [
+    {
+      kind: "ExtractiveSummarization",
+      maxSentenceCount: 2,
+    },
+  ];
+  const poller = await client.beginAnalyzeBatch(actions, documents, "en");
+  const results = await poller.pollUntilDone();
+  for await (const actionResult of results) {
+    if (actionResult.kind !== "ExtractiveSummarization") {
+      throw new Error(`Expected extractive summarization results but got: ${actionResult.kind}`);
+    }
+    if (actionResult.error) {
+      const { code, message } = actionResult.error;
+      throw new Error(`Unexpected error (${code}): ${message}`);
+    }
+    for (const result of actionResult.results) {
+      console.log(`- Document ${result.id}`);
+      if (result.error) {
+        const { code, message } = result.error;
+        throw new Error(`Unexpected error (${code}): ${message}`);
+      }
+      console.log("Summary:");
+      console.log(result.sentences.map((sentence) => sentence.text).join("\n"));
+    }
+  }
+}
+main();
+```
+
+### Abstractive Summarization
+
+Abstractive summarization generates summaries from the input articles.
```

Comment:
How is this different from extractive summarization?

Reply:
The keyword here is "generates" as opposed to "extracts". I am open to rewording suggestions though 😊

Reply:
Did we get a design doc from the service team on this one? Wording is subtle atm, hoping we can make it more obvious.

Reply:
Sadly, no

Reply:
Okay pinged the thread

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/23208/commits/2ea10eac05b65cb762eff7b69706300e71f5da99